scenario_utility/include/scenario_utility/misc.h

```diff
@@ -41,30 +41,30 @@ constexpr simulation_is operator ||(const simulation_is& lhs,
 }
 
 
-static_assert( ( simulation_is::succeeded && simulation_is::succeeded ) == simulation_is::succeeded );
```

Comment:
@mitsudome-r [static asserts requires two arguments before C++17.](https://en.cppreference.com/w/cpp/language/static_assert) To compile with C++14 without having warning and a fail compilation this requires a message - though I'm not sure what the messages should say if anything

Reply:
If something is wrong, it looks like it will still tell you where it failed so I think this is okay for not.
e.g. changing `==` to `!=` gives:
```
/home/mitsudome-r/AutowareArchitectureProposal/src/simulator/scenario_runner/scenario_utility/include/scenario_utility/misc.h:45:73: error: static assertion failed
   45 | static_assert( ( simulation_is::succeeded && simulation_is::ongoing   ) != simulation_is::ongoing  , "");
```

Reply:
Ok I will leave this as is